lib/AST/SubstitutionMap.cpp

```diff
@@ -395,6 +398,40 @@ SubstitutionMap::combineSubstitutionMaps(const SubstitutionMap &firstSubMap,
     });
 }
 
+void SubstitutionMap::verify() const {
+#ifndef NDEBUG
```

Comment:
Well, the overhead of calling an empty method should not affect the performance IMHO.